go/pkg/pwapi/api_admin-challenge-add.go

```diff
@@ -28,7 +29,20 @@ func (svc *service) AdminChallengeAdd(ctx context.Context, in *AdminChallengeAdd
 		Homepage:    in.Challenge.Homepage,
 	}
 
-	err := svc.db.Create(&challenge).Error
+	var challengeCheck pwdb.Challenge
+	err := svc.db.
+		Where(&pwdb.Challenge{
+			Slug: in.Challenge.Slug,
+		}).
+		First(challengeCheck).
```

Comment:
```suggestion
		First(&challengeCheck).
```

Reply:
btw, I strongly suggest you to use our existing helper: https://pkg.go.dev/pathwar.land/pathwar/v2/go/pkg/pwdb#GetIDBySlugAndKind